src/main/java/com/google/sps/servlets/ColumnsFromTablesServlet.java

```diff
@@ -24,13 +24,18 @@
 import com.google.cloud.spanner.Statement;
 import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.*; 
```

Comment:
Maybe avoid the use of ' * ' and only import what we need

Reply:
Ah okay I used this to import java.util.Set so I'll change that!

Reply:
Done!